util/image_viewer/spotfinder_frame.py

```diff
@@ -1161,6 +1165,48 @@ def update_settings(self, layout=True):
                     fontsize=self.settings.fontsize,
                 )
 
+        if self.settings.show_thresh_pix:
+            image = self.pyslip.tiles.raw_image
+            kabsch_debug_list = self._calculate_dispersion_debug(image)
+            cv = [kabsch.index_of_dispersion() for kabsch in kabsch_debug_list]
+            final_mask = [kabsch.final_mask() for kabsch in kabsch_debug_list]
+            final_mask = [mask.as_1d() for mask in final_mask]
+            value = []
+            for pnl, mask in enumerate(final_mask):
+                mask.reshape(cv[pnl].accessor())
```

Comment:
I'm not sure why the mask is converted to 1d and then reshaped, but it doesn't appear to be needed (I tested the change below and it seems to work):
```suggestion
            final_mask = [kabsch.final_mask() for kabsch in kabsch_debug_list]
            value = []
            for pnl, mask in enumerate(final_mask):
```

Reply:
probably copypasta